examples/Planning/ForecastReach.cs

```diff
@@ -0,0 +1,313 @@
+
+            Console.WriteLine($"Plannable Products for location {locationId}:");
+            foreach (ProductMetadata product in response.ProductMetadata)
+            {
+                Console.WriteLine($"{product.PlannableProductCode}:");
+                Console.WriteLine("Age Ranges:");
+                foreach (var ageRange in product.PlannableTargeting.AgeRanges)
+                {
+                    Console.WriteLine($"\t- {ageRange}");
+                }
+                Console.WriteLine("Genders:");
+                foreach (var gender in product.PlannableTargeting.Genders)
+                {
+                    Console.WriteLine($"\t- {gender.Type}");
+                }
+                Console.WriteLine("Devices:");
+                foreach (var device in product.PlannableTargeting.Devices)
+                {
+                    Console.WriteLine($"\t- {device.Type}");
+                }
+            }
+        }
+
+        /// <summary>
+        /// Create a base request to generate a reach forecast.
+        /// </summary>
+        /// <param name="customerId">The customer ID for the reach forecast.</param>
+        /// <param name="productMix">The product mix for the reach forecast.</param>
+        /// <param name="locationId">Location ID from somewhere returned by ListPlannableLocations.</param>
+        /// <param name="currencyCode">Three-character ISO 4217 currency code.</param>
+        public GenerateReachForecastRequest BuildReachRequest(
+                   string customerId,
+                   List<PlannedProduct> productMix,
+                   string locationId,
+                   string currencyCode)
+        {
+            CampaignDuration duration = new CampaignDuration()
+            {
+                DurationInDays = 30
+            };
+
+            GenderInfo[] genders = new GenderInfo[]
+            {
+                new GenderInfo() { Type = GenderType.Female },
+                new GenderInfo() { Type = GenderType.Male }
+            };
+
+            DeviceInfo[] devices = new DeviceInfo[]
+            {
+                new DeviceInfo() { Type = Device.Desktop },
+                new DeviceInfo() { Type = Device.Mobile },
+                new DeviceInfo() { Type = Device.Tablet }
+            };
+
+            Targeting targeting = new Targeting()
+            {
+                PlannableLocationId = locationId,
+                AgeRange = ReachPlanAgeRange.AgeRange1865Up,
+
+            };
+            targeting.Genders.AddRange(genders);
+            targeting.Devices.AddRange(devices);
+
+            GenerateReachForecastRequest request = new GenerateReachForecastRequest()
+            {
+                CustomerId = customerId,
+                CurrencyCode = currencyCode,
+                CampaignDuration = duration,
+                Targeting = targeting,
+                MinEffectiveFrequency = 3
```

Comment:
Document magic number.

Reply:
Set this back to the same default used by Reach Planner (1), and added a link to the request's reference docs.